frontend/src/metabase/timelines/questions/components/TimelineCard/TimelineCard.unit.spec.tsx

```diff
@@ -39,28 +39,14 @@ describe("TimelineCard", () => {
 
     expect(props.onToggleTimeline).toHaveBeenCalled();
   });
-
-  it("should make a timeline visible when its event is selected", () => {
```

Comment:
This test block is removed as long as we don't want to show a timeline by clicking on a hidden event in that timeline.

This is a design decision we're waiting on for now.